packages/angular/src/table/head/table-head-cell.component.ts

```diff
@@ -0,0 +1,117 @@
+import { Component, HostBinding, ViewEncapsulation } from '@angular/core';
+import { TableHeadCell } from 'carbon-components-angular';
+
+@Component({
+  // tslint:disable-next-line: component-selector
+  selector: '[aiTableHeadCell]',
+  template: `
+    <ng-container *ngIf="!skeleton">
+      <button
+        class="bx--table-sort"
+        *ngIf="this.sort.observers.length > 0 && column.sortable"
+        [attr.aria-label]="
+          (column.sorted && column.ascending ? getSortDescendingLabel() : getSortAscendingLabel())
+            | async
+        "
+        aria-live="polite"
+        [ngClass]="{
+          'bx--table-sort--active': column.sorted,
+          'bx--table-sort--ascending': column.ascending
+        }"
+        (click)="onClick()"
+      >
+        <span
+          *ngIf="!column.template"
+          class="table-head-cell-text"
+          [title]="column.data"
+          tabindex="-1"
+        >
+          {{ column.data }}
+        </span>
+        <ng-template
+          [ngTemplateOutlet]="column.template"
+          [ngTemplateOutletContext]="{ data: column.data }"
+        >
+        </ng-template>
+        <span class="table-head-cell-icons">
+          <svg
```

Comment:
Do we have icons we can use with the directive instead of dropping the raw SVG in?